packages/components/ng-ovh-payment-method/src/payment-method.constants.js

```diff
@@ -73,6 +73,7 @@ export const PAYMENT_METHOD_TYPE_ENUM = {
 };
 
 export const TYPE_INTEGRATION_ENUM = {
+  COMPONENT: 'COMPONENT',
```

Comment:
Sounds generic, is it what is provided by the adyen lib ?

Reply:
No it's a type of integration returned by /me api